app/assets/stylesheets/card.scss

```diff
@@ -1,7 +1,8 @@
 .hf-card {
   &-container {
-    background-color: govuk-colour('light-grey');
-    border-bottom: 3px solid govuk-organisation-colour('department-for-education', false);
+    background-color: govuk-colour("light-grey");
```

Comment:
It was be preferable to use single quotes in this file (where possible) to ensure there is consistency